.vib/rails/goss/vars.yaml

```diff
@@ -0,0 +1,20 @@
+binaries:
+  - gosu
+  - mysql
+  - node
+  - python
+  - rails
+  - ruby
+  - wait-for-port
```

Comment:
We usually do not add `wait-for-port` here, as we understand that it is only a minor tool used at initialization time. As such, the existence of this binary is implicitly tested when the container is fully deployed. No strong thought on this though, wdyt @joancafom?